test/e2e/getenvoy_test.go

```diff
@@ -30,6 +30,16 @@ func TestGetEnvoyVersion(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestGetEnvoyList(t *testing.T) {
+	t.Parallel()
+
+	stdout, stderr, err := getEnvoy("list").Exec()
```

Comment:
never tested this before. This helps make sure we don't goof json parsing in real life data